src/Microsoft.AspNetCore.Localization/AcceptLanguageHeaderRequestCultureProvider.cs

```diff
@@ -33,7 +34,7 @@ public override Task<ProviderCultureResult> DetermineProviderCultureResult(HttpC
 
             if (acceptLanguageHeader == null || acceptLanguageHeader.Count == 0)
             {
-                return Task.FromResult((ProviderCultureResult)null);
+                return TaskCache<ProviderCultureResult>.DefaultCompletedTask;
```

Comment:
@hishamco this is fine. Good in fact. We were cleaning up cacheable `Task`s and @kichalla got ahead of the curve.